Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -73,9 +73,48 @@ public class SchemeGenerator {
         return xcschemes
     }
 
+    private func getProject(for container: String?) -> PBXProj? {
```

Comment:
Is this function used?

Reply:
Yes you are right. Thanks. I removed that function.